testeng/jobs/upgradePythonRequirements.groovy

```diff
@@ -476,6 +476,31 @@ Map edxWhen= [
     alwaysNotify: false
 ]
 
+Map eventTracking= [
+    org: 'edx',
+    repoName: 'event-tracking',
+    targetBranch: "master",
+    pythonVersion: '3.5',
```

Comment:
This repo had issues with the newest version of celery that gets pulled in when I build requirements in python 3.8 so I'm leaving it at 3.5 for now so that arbi-bom can go through the upgrade with this when they are doing this work for the other repos.

Reply:
Is it possible to drop comments in the file around this?

Reply:
Added a comment in the file.